components/camel-direct/src/main/java/org/apache/camel/component/direct/DirectProducer.java

```diff
@@ -33,14 +33,14 @@ public DirectProducer(DirectEndpoint endpoint) {
     }
 
     @Override
+    @Deprecated
```

Comment:
start and stop methods should not be deprecated, since they dont do anything then just remove them so they just use from parent